### PR TITLE
Add docs for releasing Phoenix assets

### PR DESCRIPTION
**Pull request by micbar**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+#### Updating ocis-phoenix
+
+1. Create a branch `release-$version`. in <https://github.com/owncloud/ocis-phoenix>
+2. Create a Folder in `changelog` for the release version and date `mkdir $major.$minor.$patchVersion_YYYY-MM-DD`.
+3. Move all changelog items from the `changelog/unreleased/` folder to the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
+4. Update the go module `ocis-pkg` to the latest version <https://blog.golang.org/using-go-modules> .
+5. Update the phoenix asset in the Makefile.
+6. Run `go generate`.
+7. Create Changelog item for the Update in the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
```
Comment (micbar) [inline docs/releasing.md:45]: make generate

**Comment by micbar** [comment]
@PVince81 @LukasHirt This is laying around some time now.

Can you give Feedback?

**Comment by LukasHirt** [comment]
Looks nice, just thinking though if the docs is the right place for this. Since the releases will be anyway done by the team and it's more a process then maybe would handbook be a better location?

^ Not saying it should be like that, just asking :wink:

File: docs/releasing.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+#### Releasing Phoenix Frontend
+
+1. Create a branch `release-$version`. in <https://github.com/owncloud/phoenix>
+2. Create a Folder in `changelog` for the release version and date `mkdir $major.$minor.$patchVersion_YYYY-MM-DD`.
+3. Move all changelog items from the `changelog/unreleased/` folder to the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
+4. Commit your changes.
+5. After merging, wait for the CI to run on the merge commit.
+6. Go to "Releases" in GH click "Draft a new Release".
```
Comment (LukasHirt) [inline docs/releasing.md:33]: Missing "and" before click?

File: docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+7. Use `v$major.$minor.$patch` as a tag (the `v` prefix is important) and publish it.
+8. The tag and the Release artifacts will be created automatically.
+
+#### Updating ocis-phoenix
+
+1. Create a branch `release-$version`. in <https://github.com/owncloud/ocis-phoenix>
+2. Create a Folder in `changelog` for the release version and date `mkdir $major.$minor.$patchVersion_YYYY-MM-DD`.
+3. Move all changelog items from the `changelog/unreleased/` folder to the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
+4. Update the go module `ocis-pkg` to the latest version <https://blog.golang.org/using-go-modules> .
+5. Update the phoenix asset in the Makefile.
```
Comment (PVince81) [inline docs/releasing.md:43]: how ?

File: docs/releasing.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+6. Go to "Releases" in GH click "Draft a new Release".
+7. Use `v$major.$minor.$patch` as a tag (the `v` prefix is important) and publish it.
+8. The tag and the Release artifacts will be created automatically.
+
+#### Updating ocis-phoenix
+
+1. Create a branch `release-$version`. in <https://github.com/owncloud/ocis-phoenix>
+2. Create a Folder in `changelog` for the release version and date `mkdir $major.$minor.$patchVersion_YYYY-MM-DD`.
+3. Move all changelog items from the `changelog/unreleased/` folder to the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
+4. Update the go module `ocis-pkg` to the latest version <https://blog.golang.org/using-go-modules> .
```
Comment (PVince81) [inline docs/releasing.md:42]: how ? add the command directly

**Comment by PVince81** [comment]
@micbar can we move this to the repo ocis-phoenix ? we could still link to that page from here

I can take care if you agree

**Comment by PVince81** [comment]
to clarify: we keep the Phoenix repo related docs here in this PR but move the ocis-phoenix ones to the other repo and link them together

File: docs/releasing.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+5. Update the phoenix asset in the Makefile.
+6. Run `go generate`.
```
Comment (PVince81) [inline docs/releasing.md:44]:
```
% go generate
can't load package: package .: build constraints exclude all Go files in /home/vincent/work/workspace/ocis-phoenix
```

**Comment by PVince81** [comment]
I've extracted the ocis-phoenix bit into https://github.com/owncloud/ocis-phoenix/pull/71

next up: reducing this PR to the minimum needed for a Phoenix release

**Comment by PVince81** [comment]
rebased and adjusted, please review

**Pull request merged**